test/testhelper.js

```diff
@@ -12,7 +12,7 @@ governing permissions and limitations under the License.
 
```

Comment:
Do not move this file. It is exported as API (for downstream modules that need this utility for their own tests) and has to be in `lib/` for that.

Reply:
(It's moved)